sigar_windows.go

```diff
@@ -352,6 +347,21 @@ func (self *ProcTime) Get(pid int) error {
 	return nil
 }
 
+func getProcTimes(pid int) (*syscall.Rusage, error) {
+	handle, err := syscall.OpenProcess(processQueryLimitedInfoAccess, false, uint32(pid))
+	if err != nil {
+		return nil, errors.Wrapf(err, "OpenProcess failed for pid=%v", pid)
+	}
+	defer syscall.CloseHandle(handle)
+
+	var CPU syscall.Rusage
```

Comment:
Isn't lower camel case normally used for variable names? (e.g. `cpu`)

Reply:
I will rename it. I just moved the code without checking the format.

Reply:
Then it's probably something I wrote. 😆 It just looked weird seeing an all caps var.